config.js

```diff
@@ -76,6 +76,9 @@ module.exports = {
         return {
           destination: `build/web/scss/${filePath}.scss`,
           format: 'scss/variables',
+          options: {
+            showFileHeader: false,
```

Comment:
Reading [this code on how it generates this](https://github.com/amzn/style-dictionary/blob/17f4cb2f30bd002dfd55d6ef8c5bee4138de8d64/lib/common/formatHelpers/fileHeader.js#L48), I found a way you can overwrite it! We can lose the date but still keep the "Do not edit file directly"

First set the header to a string var...
```javascript
const fileHeaderText = `Do not edit directly`;
```

Then apply this change:

```suggestion
            fileHeader: () => [fileHeaderText],
```

It will generate:
```
/**
 * Do not edit directly
 */
```

What do you think?

Reply:
@daine great idea, I updated the PR!

Reply:
Yay thank you!